middleware/authentication.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var config = require('../config');
+
+const methodToCatch = ['PUT', 'DELETE', 'POST'];
+
+module.exports = (req, res, next) => {
+    if (methodToCatch.indexOf(req.method) !== -1) {
```

Comment:
Why do you want to only catch these methods? We will maybe have some `GET` endpoints that we will protect.